### ISSUE TITLE
About special token implementation

### ISSUE THREAD

**Issue by SchenbergZY** [CyberZHG/keras-gpt-2#9]
Is there a way to make special token like BERT model for classification? For example, making something like <pad> or <cls> instead of padding with 0s at the end of the text? Because the 0 in the vocabulary means "!" but not <pad>/

**Comment by CyberZHG** [comment]
Actually, it's not necessary since the model isn't bi-directional. Consider using `keras.layers.Masking` and add it to the output to support masking.

**Comment by SchenbergZY** [comment]
So you mean I can add a masking to mask the padding positions for each samples, right?

**Comment by SchenbergZY** [comment]
Anyway thank you for your reply and I reach some good CV results in my own task.
